Resonans/Views/Tools/ToolItems/BgRemover/RemoveBackground/RemoveBackgroundView.swift

```diff
@@ -1,137 +1,116 @@
-//
-//  RemoveBackgroundView.swift
-//  Resonans
-//
-//  Created by Kevin Dallian on 25/10/25.
-//
-
 import SwiftUI
+import UIKit
 
 struct RemoveBackgroundView: View {
-    @StateObject var viewModel: RemoveBackgroundViewModel
-    
+    @StateObject private var viewModel: RemoveBackgroundViewModel
     @Environment(\.dismiss) private var dismiss
-    @Environment(\.colorScheme) private var colorScheme
+
+    @State private var activeSheet: ActiveSheet?
+
     @AppStorage("accentColor") private var accentRaw = AccentColorOption.purple.rawValue
-    @State var activeSheet: ActiveSheet?
-    private var accent: AccentColorOption { AccentColorOption(rawValue: accentRaw) ?? .purple }
-    
+
+    private var accentColor: Color { (AccentColorOption(rawValue: accentRaw) ?? .purple).color }
+
     init(image: UIImage) {
-        self._viewModel = StateObject(wrappedValue: RemoveBackgroundViewModel(image: image))
+        _viewModel = StateObject(wrappedValue: RemoveBackgroundViewModel(image: image))
     }
-    
+
     var body: some View {
-        VStack {
-            headerSection
-            imageSection
-            Spacer()
-            footerButton
-        }
-        .presentationDetents([.medium])
-        .padding(.top, 12)
-        .padding(.horizontal, 24)
-        .background(
-            LinearGradient(
-                colors: [accent.gradient, colorScheme == .dark ? .black : .white],
-                startPoint: .topLeading,
-                endPoint: .bottom
-            )
-            .ignoresSafeArea()
-        )
-        .onChange(of: viewModel.errorMessage) { oldError, newError in
-            if let newError, oldError != newError, !newError.isEmpty {
-                activeSheet = .removeFailed(errorMessage: newError)
+        NavigationStack {
+            Form {
+                Section("Image") {
+                    VStack(alignment: .center, spacing: 12) {
+                        ResizableImage(image: viewModel.image)
+                        Text("Preview shows the exact image that will be processed.")
+                            .font(.footnote)
+                            .foregroundStyle(.secondary)
+                            .multilineTextAlignment(.center)
+                    }
+                    .frame(maxWidth: .infinity, alignment: .center)
+                }
+
+                Section {
+                    Button(action: viewModel.removeBackground) {
+                        if viewModel.isLoading {
+                            HStack {
+                                ProgressView()
+                                Text("Removing background…")
+                            }
+                        } else {
+                            Text("Remove background")
+                        }
```

Comment:
This causes compiler error on Xcode 16.
<img width="987" height="266" alt="Screenshot" src="https://github.com/user-attachments/assets/c717f5fe-b8d4-41d9-a666-090e5ac4766b" />
Try this
```suggestion
Group {
    if viewModel.isLoading {
        HStack {
            ProgressView()
            Text("Removing background…")
        }
    } else {
        Text("Remove background")
    }
}
```